charts/enterprise-metrics/templates/graphite-querier-schemas-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.graphite.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "enterprise-metrics.fullname" . }}-graphite-schemas
+  labels:
+    app: {{ template "enterprise-metrics.name" . }}
+    chart: {{ template "enterprise-metrics.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  storage-schemas.conf: |
+    [default]
+    pattern = .*
+    intervals = 0:1s
+    retentions = 10s:8d,10min:1y
```

Comment:
should we make this configurable (with this value as default)?

Reply:
I was not 100% sure about this because I don't see these values changing unless the amount of metrics is very high but you're right lets just make it configurable from the beginning. Addressed here https://github.com/grafana/helm-charts/pull/872/commits/e40f53b2a5ebac59d58c562670244104adeaf7c4